<launch>
    <!-- 定义命令行参数 -->
    <arg name="bag_file" default="ros_bag_file_path" />

    <node pkg="teach_pendant" type="teach_pendant.py" name="teach_pendant_node" args=" --play $(arg bag_file) " output="screen">
    </node>
</launch>